### DISCUSSION TITLE
AYUDA - CHAMILO 1.11.26 NO PERMITE VISUALIZAR EL PROGRESO DE MIS ESTUDIANTES

### DISCUSSION THREAD

**Discussion by SoyLusho28** [chamilo/chamilo-lms#5765] [General]
Buen dia para todos.

Tengo un problema en la versión 1.11.26 de chamilo, no sé por que ocurre, en el modulo de "mi progreso" para el perfil de estudiante, no les permite visualizar ninguna información a pesar de estos ya hayan realizado varias capacitaciones. ¿Alguien tiene conocimiento del porque pasa esto?

¡Muchas gracias!

![1](https://github.com/user-attachments/assets/42d9c953-c03c-4b71-9c99-0d8afeb6344e)

**Comment by Kaneda-1** [comment]
Hola. 

¿Has comprobado si los estudiantes siguen inscritos en los cursos?
¿Esos cursos están visibles?

Un saludo.

**Reply by SoyLusho28** [reply]
Hola!
Si, de hecho los estudiantes son inscritos por medio de "clases", y el estado del curso es "privado" solo para que ellos no se puedan inscribir si no que sea el profesor el que lo haga

**Comment by chatonito** [comment]
Hola:
no sé si  sea el caso, el progreso de los estudiantes se ve  en las lecciones y su contenido. si en las lecciones tienes lecturas o progreso en las mismas, esta se verá en el reporte, de acuerdo al curso y a la sesion, incluso si envió las tareas.
te dejo un pantallazo, que espero ayude en algo.
saludos
![Captura de Pantalla 2024-09-09 a la(s) 10 59 19](https://github.com/user-attachments/assets/07f3a51a-2572-4195-bcf3-6142305a2d7d)
